Changelog.md

```diff
@@ -14,6 +14,7 @@ Bugfixes:
  * Code Generator: Fix internal compiler error when calling functions bound to calldata structs and arrays.
  * Code Generator: Fix internal compiler error when passing zero to ``bytes.concat()``.
  * Type Checker: Fix internal error and prevent static calls to unimplemented modifiers.
+ * Yul Code Generator: Fix internal compiler error when using a long literal with binary negation.
```

Comment:
```suggestion
 * Yul Code Generator: Fix internal compiler error when using a long literal with bitwise negation.
```

Reply:
Actually didn't we say we don't put changelog entries for the ir generator?

Reply:
Ah, right. Removing..